commit/factory.go

```diff
@@ -27,9 +27,35 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
 )
 
-// maxQueryLength is set to twice the maximum size of a theoretical merkle root processor query
-// that assumes 1,000 source chains and 256 (theoretical max) RMN nodes.
-const maxQueryLength = 615_520
+const (
+	// Estimated maximum number of source chains the system will support.
+	// This value should be adjusted as we approach supporting that number of chains.
+	// Its primary purpose is to assist in defining the limits below.
+	estimatedMaxNumberOfSourceChains = 900
+
+	// Estimated maximum number of RMN nodes the system will support.
+	estimatedMaxRmnNodesCount = 256
+
+	// Estimated maximum number of priced tokens that the Commit DON supports.
+	// This value does not indicate a system limitation but just an estimation to properly tune the OCR parameters.
+	// The value can be adjusted as needed.
+	estimatedMaxNumberOfPricedTokens = 10_000
+
+	// maxQueryLength is set to twice the maximum size of a theoretical merkle root processor query
+	// that assumes 1,000 source chains and 256 (theoretical max) RMN nodes.
```

Comment:
```suggestion
	// that assumes 900 source chains and 256 (theoretical max) RMN nodes.
```

Should be 900 since you updated the above. Or maybe reference `estimatedMaxNumberOfSourceChains` directly